Cargo.toml

```diff
@@ -19,7 +19,7 @@ prost-derive = "0.5"
 bytes = "0.4"
 futures = "0.1"
 grpcio = { version = "0.5.0-alpha.1", features = ["prost-codec"] }
-raft = { git = "https://github.com/pingcap/raft-rs.git", rev = "8e953b0d0c9cd2f98c94a42edc48ae3f09e967ff" }
+raft-proto = "0.4"
```

Comment:
Can you explain the versioning strategy you expect?

Reply:
By following raft-rs?

Reply:
Yes, matching with Raft. (Although if we do something different, it doesn't matter much).